tests/shell/shell_recorder/tutorial_wrapper/SyntaxCheck.md

```diff
@@ -0,0 +1,39 @@
+Test1
+```sh
+# Mountpoint:/test
```

Comment:
Mountpoint is needed for the "normal" shellrecorder (check for file/database changes and backup/restoring of key below that mountpoint)